app/liquid_tags/link_tag.rb

```diff
@@ -1,52 +1,32 @@
 class LinkTag < LiquidTagBase
   include ActionView::Helpers
-  attr_reader :article
+  PARTIAL = "articles/liquid".freeze
 
   def initialize(_tag_name, slug_or_path_or_url, _tokens)
-    @article = parse_url_for_article(slug_or_path_or_url)
+    @article = get_article(slug_or_path_or_url)
+    @title = strip_tags @article.title
```

Comment:
nit: while we don't have a strict rule, it's a good idea to stay consistent.
```suggestion
    @title = strip_tags(@article.title)
```

Reply:
updated with your suggested change.